prow/pilot-e2e-smoketest.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+
+# Copyright 2017 Istio Authors
+
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+
+#       http://www.apache.org/licenses/LICENSE-2.0
+
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+
+
+#######################################
+# Smoketest script triggered by Prow. #
+#######################################
+
+# Exit immediately for non zero status
+set -e
+# Check unset variables
+set -u
+# Print commands
+set -x
+
+if [ "${CI:-}" == "bootstrap" ]; then
+    # Use the provided pull head sha, from prow.
+    GIT_SHA="${PULL_PULL_SHA}"
+else
+    # Use the current commit.
+    GIT_SHA="$(git rev-parse --verify HEAD)"
+fi
+
+echo "=== Clone istio/istio ==="
+ISTIO_TMP_DIR="$(mktemp -d istio-XXXXX)"
+git clone --depth 1 https://github.com/istio/istio "./${ISTIO_TMP_DIR}"
+cd "./${ISTIO_TMP_DIR}"
+
+HUB="gcr.io/istio-testing"
+BUCKET="istio-artifacts"
+ISTIOCTL_URL="https://storage.googleapis.com/${BUCKET}/pilot/${GIT_SHA}/artifacts/istioctl"
+ARTIFACTS_DIR="${GOPATH}/src/istio.io/pilot/_artifacts"
```

Comment:
I think you should use github.com/istio instead of istio.io as you did not create the link. You know if it works when you have an artifact folder like in 
https://pantheon.corp.google.com/storage/browser/istio-prow/pull/istio_istio/470/istio-presubmit/116/